src/orion/core/cli/base.py

```diff
@@ -71,6 +74,7 @@ def parse(self, argv):
             format="%(asctime)-15s::%(levelname)s::%(name)s::%(message)s",
             level=levels.get(verbose, logging.DEBUG),
         )
+        logger.debug("Orion version : " + orion.core.__version__)
```

Comment:
You have an error here: https://github.com/Epistimio/orion/runs/4939592554?check_suite_focus=true#step:5:15
This is because pylint prefers a format with strings using '%s' instead of concatenating them. So it should rather be

```python
logger.debug("Orion version : %s", orion.core.__version__)
```

Sorry I didn't catch this in the first place. :sweat_smile: